src/components/Typography/index.tsx

```diff
@@ -18,7 +18,15 @@ export const Headline: FC<HeadingProps> = (props) => {
 }
 
 export const H1: FC<TextProps> = (props) => {
-  return <Text as="h1" fontSize="6xl" lineHeight="64px" {...props} />
+  return (
+    <Text
+      as="h1"
+      fontSize="60px"
```

Comment:
started a discussion on this with sasha and liz: https://discord.com/channels/893441201628405760/893457138079399936/951230101801025696